components/cors-mgt/org.wso2.carbon.identity.cors.mgt.core/src/main/java/org/wso2/carbon/identity/cors/mgt/core/CORSManagementService.java

```diff
@@ -53,14 +54,33 @@
      * @param corsOrigins  A list of CORS Origins to be added.
      * @throws CORSManagementServiceException
      */
-    void addCORSOrigins(String tenantDomain, List<CORSOrigin> corsOrigins) throws CORSManagementServiceException;
+    void addCORSOrigins(String tenantDomain, List<Origin> corsOrigins) throws CORSManagementServiceException;
 
     /**
      * Delete the CORS Origin(s) from the existing CORS Origin list of the tenant.
      *
      * @param tenantDomain The tenant domain.
-     * @param corsOrigins  A list of CORS Origins to be deleted.
+     * @param ids          A list of CORS Origin IDs to be deleted.
      * @throws CORSManagementServiceException
      */
-    void deleteCORSOrigins(String tenantDomain, List<CORSOrigin> corsOrigins) throws CORSManagementServiceException;
+    void deleteCORSOrigins(String tenantDomain, List<String> ids) throws CORSManagementServiceException;
```

Comment:
Since you have changed a parameter type of an already existing method, won't this break the existing implementations?

Reply:
Nothing uses this service yet :smile: